src/digisos/skjema/familie/forsorgerplikt/ForsorgerPlikt.tsx

```diff
@@ -53,6 +55,8 @@ const ForsorgerPliktView = () => {
                 <p>
                     <FormattedHTMLMessage id="familierelasjon.ingen_registrerte_barn" />
                 </p>
+                {/*<BrukerregistrerteBarn/>*/}
```

Comment:
Kanskje en liten kommentar for hvorfor den er kommentert ut? 
`Disablet fram til alle fagsystem kan håndtere brukerregistrerte barn. `